scripts/test-atypical-padding.py

```diff
@@ -422,6 +451,8 @@ def main():
     node = node.add_child(ClientHelloGenerator(ciphers, extensions=ext))
     extensions = {ExtensionType.encrypt_then_mac:None,
```

Comment:
This part is a bit confusing when there is `ext` couple of lines above and `extensions` here. What about using something like `ext_ch` and `ext_sh` for example? (Relevant in more places.)

Reply:
I'll leave `ext` as-is, as that's what we use basically everywhere for client hello, but good point on `extensions`, changed it to `srv_ext`, as it is in other scripts